src/workerd/api/tests/commonjs-module-test.wd-test

```diff
@@ -0,0 +1,18 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "commonjs-module-test",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "commonjs-module-test.js"),
+          (name = "foo",
+           commonJsModule = "exports.foo = 1",
+           namedExports = ["foo", "bar"])
```

Comment:
interesting. I think this will work well enough. 99.999% of developers will never need this as we'll generate these on the fly and provide it via fallback service.